go/go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/valkey-io/valkey-glide/go/glide
 
-go 1.18
+go 1.20
```

Comment:
Why?

Reply:
In order to optimize we need API that requires `unsafe.Slice` + `unsafe.StringData` - both require v1.20.
We checked (@barshaul  and I) and it is common to support 2 last `go` versions (see for example `go-redis`).